bin/i18n/resources/apps/musiclab_libraries/sync_in.rb

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env ruby
+
+require 'fileutils'
+require 'json'
+
+require_relative '../../../i18n_script_utils'
+require_relative '../../../utils/sync_in_base'
+require_relative '../musiclab_libraries'
+
+module I18n
+  module Resources
+    module Apps
+      module MusiclabLibraries
+        class SyncIn < I18n::Utils::SyncInBase
+          def process
+            LIBRARY_NAME_IN_OUT_MAPPINGS.each do |name_map|
+              I18nScriptUtils.write_json_file(
+                CDO.dir(I18N_SOURCE_DIR, DIR_NAME, "#{name_map[:to]}.json"),
+                get_translation_strings("#{name_map[:from]}.json")
+              )
+            end
+          end
+
+          private def get_translation_strings(library_filename)
+            manifest = JSON.parse(
+              AWS::S3.download_from_bucket(
+                'cdo-curriculum',
+                "media/musiclab/#{library_filename}"
+              )
+            )
+
+            strings_to_translate = {}
+
+            manifest['instruments'].each do |instrument|
```

Comment:
I should note that a vast majority of the changes in this PR are boilerplate/tests/largely inspired by the animation manifest translation approach -- this (hopefully not too complicated) bit is the main logic that's new.

Reply:
Not necessary for this PR, but it could be interesting to think about how much could be factored into common code, especially if we decided to use S3 to store translations for other situations.